dev/drivers/scripts/rtofs/stats/jevs_rtofs_argo_stats.sh

```diff
@@ -43,6 +29,8 @@ export VERIF_CASE=grid2obs
 export VARS="temp psal"
 export COMPONENT=rtofs
 
+source $HOMEevs/modulefiles/${COMPONENT}/${COMPONENT)_${STEP}.sh
```

Comment:
One small typo that needs fixing ... the closing brace for the second COMPONENT needs to be curly.

Reply:
Done.